automatic/mpc-hc-clsid2/update.ps1

```diff
@@ -27,8 +27,7 @@ function global:au_GetLatest {
   $re = 'x86\.exe$'
   $url32 = $download_page.Links | ? href -match $re | select -first 1 -expand href | % { 'https://github.com' + $_ }
 
-  $re = 'x64\.exe$'
-  $url64 = $download_page.links | ? href -match $re | select -first 1 -expand href | % { 'https://github.com' + $_ }
+  $url64 = $url32 -replace "x86","x64"
```

Comment:
Do not rely on blindly replacing `x86` with `x64`.
There have been times in the past where the naming haven't been exact.

TBH, this change looks entirely unnecessary.
Please explain why this change is necessary.

Reply:
I see know why you made that change.
However, my first two lines still stands as the name isn't guaranteed to be the same.

Please only update the regex to also check the version instead.

_Remember to always update the commit message as mentioned in your other message_